deployment/constructor_params/tapir/child.yml

```diff
@@ -3,27 +3,28 @@ deployment:
   chain_id: 80001
 
 artifacts:
-    dir: ./deployment/artifacts/
-    filename: tapir.json
+  dir: ./deployment/artifacts/
+  filename: tapir.json
 
 contracts:
-    - MockPolygonChild
-    - ProxyAdmin
-    - TapirTACoChildApplication:
-        _rootApplication: $MockPolygonChild
-    - TransparentUpgradeableProxy:
-        _logic: $TapirTACoChildApplication
-        admin_: $ProxyAdmin
-        _data: $bytes:0x
-    - TapirRitualToken:
-        _totalSupplyOfTokens: 10000000000000000000000000
-    - Coordinator:
-        _application: $TransparentUpgradeableProxy:TapirTACoChildApplication
-        _timeout: 3600
-        _maxDkgSize: 32
-        _admin: $deployer
-        _currency: $TapirRitualToken
-        _feeRatePerSecond: 1
-    - GlobalAllowList:
-        _coordinator: $Coordinator
-        _admin: $deployer
+  - MockPolygonChild
+  - ProxyAdmin
+  - TapirTACoChildApplication:
+      _rootApplication: $MockPolygonChild
+      _minimumAuthorization: 40000000000000000000000
+  - TransparentUpgradeableProxy:
+      _logic: $TapirTACoChildApplication
+      admin_: $ProxyAdmin
```

Comment:
it looks like the function signature of `TransparentUpgradeableProxy` changed [in v5.0.0.](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/932fddf69a699a9a80fd2396fd1a2ab91cdda123/contracts/proxy/transparent/TransparentUpgradeableProxy.sol#L77C41-L77C53)

```solidity
constructor(address _logic, address initialOwner, bytes memory _data) payable ERC1967Proxy(_logic, _data) {
```